### PR TITLE
Xs 3640

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Updated resource files for DQC rules 1, 13, and 15.
- Updated test cases to match the version of the code.

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [x] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/resources/DQC_US_0013/dqc_13_concepts.csv [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+2785,EffectiveIncomeTaxRateReconciliationRepatriationOfForeignEarnings
+2871,EffectiveIncomeTaxRateReconciliationTaxHolidays
+2872,EffectiveIncomeTaxRateReconciliationNondeductibleExpenseResearchAndDevelopment
+2873,EffectiveIncomeTaxRateReconciliationDeductions
+2874,EffectiveIncomeTaxRateReconciliationNondeductibleExpenseImpairmentLosses
+3001,EffectiveIncomeTaxRateReconciliationDeductionsEmployeeStockOwnershipPlanDividends
+3002,EffectiveIncomeTaxRateReconciliationNondeductibleExpenseMealsAndEntertainment
+3439,EffectiveIncomeTaxRateReconciliationDeductionsExtraterritorialIncomeExclusion
+3440,EffectiveIncomeTaxRateReconciliationNondeductibleExpenseCharitableContributions
+3441,EffectiveIncomeTaxRateReconciliationRepatriationForeignEarningsJobsCreationActOf2004
```
Comment (jasonleinbach-wf) [inline dqc_us_rules/resources/DQC_US_0013/dqc_13_concepts.csv]: Newline

Comment (allenbross-wf) [inline dqc_us_rules/resources/DQC_US_0013/dqc_13_concepts.csv]: Fixed.  Thanks!

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by joyreistad-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1 to code changes

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+10 - passes integration test cases.  Merging.

**Pull request merged**